tardis/plasma/properties/atomic.py

```diff
@@ -467,6 +467,14 @@ def calculate(self, atomic_data):
         level_idxs2line_idx = pd.Series(
             np.arange(len(index)), index=index, name="lines_idx"
         )
+
+        # Check for duplicate indices
```

Comment:
We already get duplicate indices here, completely unrelated to the actual continuum lines. For now I added a ``ValueError`` to catch this early, but I'm not sure if we can simply drop the duplicate indices.